{
  ExceptionMessage: Cannot access value of failed result
}